charts/event-handler/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for event-handler.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+#
+# Plugin specific options
+#
+teleport:
+  address: ""
+  identitySecretName: ""
+  identitySecretPath: "auth_id"
+
+eventHandler:
+  storagePath: "/var/lib/teleport/plugins/event-handler/storage"
+  timeout: "10s"
+  batch: 20
+  namespace: "default"
+
+fluentd:
+  url: ""
+  sessionUrl: ""
+  certificate:
+    secretName: ""
+    caPath: "ca.crt"
+    certPath: "client.crt"
+    keyPath: "client.key"
+
+#
+# Deployment
+#
+image:
+  repository: quay.io/gravitational/teleport-plugin-event-handler
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
```

Comment:
is this just a leftover of the skaffolding or is it the correct gid from the `teleport-plugin-event-handler`image?